doc/source/formats/index.rst

```diff
@@ -2,8 +2,34 @@
 .. _formats-label:
 
 ====================
-Formats
+Format overview
 ====================
 
-So many Formats
+MDAnalysis can read topology or coordinate information from a wide variety of file formats. The emphasis is on formats used in popular computational chemistry software. By default, MDAnalysis figures out formats by looking at the extension, unless the format is :ref:`explicitly specified <universe-loading-label>` with the ``format`` or ``topology_format`` keywords.
```

Comment:
computational chemistry --> simulation packages

Comp. chem. is too narrow – for instance, I don't see myself primarily doing "comp chem".